google/cloud/spanner/value.cc

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// To avoid secure warnings on Windows
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Sorry, but I don't know much about this macro. What warnings is this disabling on windows? Is it something we *must* disable, or are there other ways that we should address those warnings?

Reply:
It complains as follows:
```
'scanf': This function or variable may be unsafe. Consider using scanf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
```

Reply:
Here and above.  I think we can disable the warning for specific files in the CMake code instead:

https://github.com/googleapis/google-cloud-cpp/blob/84772e182e5a82c9f839a9073cef88b9962c48cf/google/cloud/storage/CMakeLists.txt#L243-L251